generator.rb

```diff
@@ -0,0 +1,112 @@
+def is_vowel?(letter)
+  vowel_array = ["a", "e", "i", "o", "u"]
+  if vowel_array.include? letter.downcase
+    # if letter == "a" || letter == "e" || letter == "i" || letter == "o" || letter == "u"
+    return true
+  else
+    return false
+  end
+end
+
+def run_generator
+  print "Please enter the first word: "
+  wordA = gets.chomp
```

Comment:
in Ruby naming conventions, this probably looks more like `word_a` compared to `wordA`. Same for `word_b`